src/pages/learn/performance.mdx

```diff
@@ -0,0 +1,81 @@
+# Performance
+
+<p className="learn-subtitle">Optimize the execution and delivery of GraphQL responses</p>
+
+At first glance, GraphQL requests may seem challenging to cache given that the API is served through a single endpoint and you may not know in advance what fields a client will include in an operation.
+
+In practice, however, GraphQL is as cacheable as any API that enables parameterized requests, such as a REST API that allows clients to specify different query parameters for a particular endpoint. There are many ways to optimize GraphQL requests on the client and server sides, as well as in the transport layer, and different GraphQL server and client libraries will often have common caching features built directly into them.
+
+On this page, we'll explore several different tactics that can be leveraged in GraphQL clients and servers to optimize how data is fetched from the API.
+
+## Client-side caching
+
+There is a range of client-side caching strategies that GraphQL clients may implement to help not only with performance but also to ensure that you render a consistent and responsive interface to your users. [Read more about client-side caching](/learn/caching).
```

Comment:
@benjie I accepted the change but I'm not sure what the "please edit" note was in reference to here.

Reply:
Just inviting an edit in case it was needed for consistency - e.g. with how you styled other links/etc throughout the docs. If you're happy with it, no change needed.